src/Library/demos/Level Bars/main.blp

```diff
@@ -0,0 +1,130 @@
+using Gtk 4.0;
+using Adw 1;
+Adw.Clamp{
+  Box {
+    orientation: vertical;
+    margin-top: 24;
+
+    Label {
+      label: "Level Bars";
+      margin-top: 6;
+      margin-bottom: 6;
+
+      styles [
+        "title-1",
+      ]
+    }
+
+    Label {
+      label: _("A bar widget used as a level indicator");
+      margin-top: 6;
+      margin-bottom: 24;
+    }
+
+    Box {
+      orientation: vertical;
+      vexpand: true;
+      spacing: 18;
+      margin-start: 18;
+      margin-end: 18;
+
+      Label {
+        label: _("Continuous Level Bar");
+      }
+
+      Box{
+        spacing:18;
+        halign: start;
+
+        Label {
+          label: _("Current Battery Level");
+        }
+
+        SpinButton spin_button{
+          digits:1;
+          adjustment: Gtk.Adjustment {
+            lower: 0;
+            upper: 100;
+            value:100;
+            step-increment: 5;
+          };
+        }
+      }
+
+      Box {
+        orientation: vertical;
+        spacing:12;
+
+        LevelBar bar_continuous{
+            margin-end:12;
+            margin-start:12;
+            margin-top:24;
+            min-value:0;
+            max-value:100;
+            mode: continuous;
+            value: bind spin_button.value;
+        }
```

Comment:
can you add the mode explicitly? even if it's not required / default

Reply:
I have? Unless you mean something else. I can move this up ``mode: continuous`` to make it the first property if you want.

Reply:
Oops, i missed it.